x-pack/plugins/observability/public/embeddable/slo/alerts/components/slo_alerts_table.tsx

```diff
@@ -66,25 +75,34 @@ export const useSloAlertsQuery = (slos: SloItem[], timeRange: TimeRange) => {
           },
           {
             bool: {
-              should: slos.map((slo) => getSloInstanceFilter(slo.id, slo.instanceId)),
+              should: slos.map((slo) =>
+                getSloInstanceFilter(slo.id, slo.instanceId, showAllGroupByInstances)
```

Comment:
Related to my comment above, maybe we don't need to send showAllGroupByInstances at all?